_posts/2021-01-07-writing-phpipam-ansible-modules.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: How to write new phpipam modules
+subtitle: Let's get in touch!
+tags: [project, phpipam, ansible, contributing]
+author: cmeissner
+---
+
+We started to develop our [phpipam-ansible-modules](https://github.com/codeaffen/phpipam-ansible-modules) to manage phpIPAM installation from within Ansible.
+We provide half a dozen of modules but unfortunately we currently does not support all controllers.
```

Comment:
We provide half a dozen of modules but unfortunately we currently do not support all controllers.